frontend/src/index.css

```
@@ -11,6 +11,18 @@
+
+  /* coheChat Theme Colors - Coffee Shop Warm Tones */
+  --cohe-primary: #8B6914;
+  --cohe-primary-dark: #6B5010;
+  --cohe-secondary: #A67C52;
+  --cohe-bg-light: #F5F0E8;
+  --cohe-bg-warm: #E8DFD0;
+  --cohe-bg-hero: #D4C4A8;
+  --cohe-text-dark: #3D3D3D;
+  --cohe-text-light: #FFFFFF;
+  --cohe-card-bg: #FFFFFF;
+  --cohe-card-shadow: rgba(139, 105, 20, 0.1);
```

Stylelint (17.2.0)

[error] Expected "0.1" to be "10%" (alpha-value-notation)

(alpha-value-notation)